numba_rvsdg/core/datastructures/scfg.py

```diff
@@ -527,14 +527,14 @@ def insert_block(
         # TODO: needs a diagram and documentaion
         # initialize new block
         new_block = block_type(
-            name=new_name, _jump_targets=tuple(successors), backedges=tuple()
+            name=new_name, _jump_targets=list(successors), backedges=[]
```

Comment:
```suggestion
            name=new_name, _jump_targets=successors, backedges=[]
```

`successors` is already a list when it comes into this function, so it doesn't need to be cast to such.